### DISCUSSION TITLE
run_model_bpa - Semantic Model

### DISCUSSION THREAD

**Discussion by Vijayinder** [microsoft/semantic-link-labs#344] [General]
Hi Team, I am running below code labs.run_model_bpa(dataset="XXXX", workspace="YYYY") this works but when I add in extended=True I get error message

I am owner of this dataset. Can someone please point me in right direction. 

labs.run_model_bpa(dataset="XXXX", workspace="YYYY", extended=True)

File /nfs4/pyenv-ed5ee546-c4ab-454d-afea-0d716a42790e/lib/python3.10/site-packages/sempy/_utils/_log.py:348, in mds_log.<locals>.get_wrapper.<locals>.log_decorator_wrapper(*args, **kwargs)
    345 start_time = time.perf_counter()
    347 try:
--> 348     result = func(*args, **kwargs)
    350     # The invocation for get_message_dict moves after the function
    351     # so it can access the state after the method call
    352     message.update(extractor.get_completion_message_dict(result, arg_dict))

OperationException: Failed to save modifications to the server. Error returned: 'It looks like you can't make changes to this dataset. Please contact a capacity administrator to make sure XMLA read/write support is enabled in the capacity settings on the Power BI Premium capacity, and then try again. For additional information, see 'XMLA read/write support' in the product documentation.

Technical Details:
RootActivityId: ab718200-c579-4f06-91de-2e21e01fdabf
Date (UTC): 12/11/2024 7:17:12 AM
'.
   at Microsoft.AnalysisServices.Tabular.Model.SaveChangesImpl(SaveFlags flags, Int32 maxParallelism)
   at Microsoft.AnalysisServices.Tabular.Model.SaveChanges()

**Comment by m-kovalsky** [comment]
I’ve actually already made a change which should solve this issue. It will be released in the next version.

**Reply by Vijayinder** [reply]
Thanks for quick response, do you have rough date when will you release it?

**Comment by m-kovalsky** [comment]
Don't have a date but it's coming soon.
